cfpb/consumerfinance.gov
# VideoPlayer: Add video timestamp

## Additions

- Add video duration timestamp in lower-right corner of videos.
- Add `formatTimestamp` utility for converting a number of seconds into a timestamp string.

## How to test this PR

1. Pull branch and run `gulp clean && gulp build`
2. Visit a video page, such as http://localhost:8000/consumer-tools/everyone-has-a-story/leah-repaying-student-loans/, http://localhost:8000/consumer-tools/reverse-mortgages/, http://localhost:8000/coronavirus/avoiding-scams/
3. See that when the video loads, a timestamp appears in its lower-right. 


## Screenshots

<img width="745" alt="Screen Shot 2021-02-11 at 10 13 57 AM" src="https://user-images.githubusercontent.com/704760/107659824-96071a80-6c55-11eb-9af2-f9ce83893363.png">

<img width="791" alt="Screen Shot 2021-02-11 at 10 15 45 AM" src="https://user-images.githubusercontent.com/704760/107659839-9a333800-6c55-11eb-9ffe-4af1659aec82.png">

<img width="885" alt="Screen Shot 2021-02-11 at 10 16 13 AM" src="https://user-images.githubusercontent.com/704760/107659841-9acbce80-6c55-11eb-99d7-c579d5192325.png">
